pkg/codegen/operations.go

```diff
@@ -304,6 +316,21 @@ func OperationDefinitions(swagger *openapi3.Swagger) ([]OperationDefinition, err
 				Spec:         op,
 			}
 
+			if opDef.OperationId == "" {
+				opDef.OperationId = strings.Title(strings.ToLower(opDef.Method))
+				for _, p := range strings.Split(opDef.Path, "/") {
```

Comment:
I see what this is fixing. We need an operationId. However, I think it's better to error out early in code generation rather than infer what the user meant. It's an executive decision - operationId is mandatory. This lets users name each operation in a meaningful way, which we can't really infer from the path.